src/DEA/allDEA.R

```diff
@@ -1,20 +1,92 @@
+#### DEA function ####
```

Comment:
Great that it's a function! This will for sure help us with generalizing to other datasets (ie TCGA, CCLE, etc).

Since the function is composed of several parts, you would want to write up a docstring at the top of the function, similar to your MATLAB scripts. Typically, I write:
  * Title of function
  * 1 line description for high level functionality
  * If composed of several modules, a paragraph or list of the various operations it does
  * List of inputs and outputs with their specified data type (ie list, data.frame, etc).

You can also use the tag @author to indicate that you wrote this up, as it is a nice function!

Reply:
Will do!